ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py

```diff
@@ -77,63 +92,69 @@ class HL2SSROSBridge(Node):
 
     def __init__(self):
         super().__init__(self.__class__.__name__)
-
-        # Declare ROS topics
-        self._image_topic = (
-            self.declare_parameter("image_topic", "")
-            .get_parameter_value()
-            .string_value
-        )
-        self._hand_pose_topic = (
-            self.declare_parameter("hand_pose_topic", "")
-            .get_parameter_value()
-            .string_value
-        )
-        self._audio_topic = (
-            self.declare_parameter("audio_topic", "")
-            .get_parameter_value()
-            .string_value
-        )
-        self.ip_addr = (
-            self.declare_parameter("ip_addr", "")
-            .get_parameter_value()
-            .string_value
-        )
-        self.pv_width = (
-            self.declare_parameter("pv_width", 1280)
-            .get_parameter_value()
-            .integer_value
-        )
-        self.pv_height = (
-            self.declare_parameter("pv_height", 720)
-            .get_parameter_value()
-            .integer_value
-        )
-        self.pv_framerate = (
-            self.declare_parameter("pv_framerate", 30)
-            .get_parameter_value()
-            .integer_value
+        self.log = self.get_logger()
```

Comment:
Possibly yes, but I wasn't sure if the logging instance returned by `get_logger()` was thread-safe.

Reply:
Oh, hadn't thought of that... I went ahead and reverted that.